### DISCUSSION TITLE
Error de version

### DISCUSSION THREAD

**Discussion by valen715** [vercel/community#3559] [Help] [answered]
### Summary

I get this error and it already makes the version change both in vercel and in the project, HELP ME PLEASE!!!!

### Steps to Reproduce

 **Node.js Version "12.x" is discontinued and must be upgraded. Please set Node.js Version to 18.x in your Project Settings to use Node.js 18.**

**Comment by swarnava** [comment, marked as answer]
Hi,

You can visit Project Settings to change Node version to 18.x. 

<img width="1422" alt="image" src="https://github.com/vercel/community/assets/1162991/0c434720-fd22-442a-8291-96147352d801">

**Reply by valen715** [reply]
I've already done that step and I still get the same error

**Reply by swarnava** [reply]
You must have *engines* property set in your package.json. If thats the case, please make sure it’s set to just 18.x without any *>*

**Reply by MMMikeM** [reply]
```json
  "engines": {
    "node": "18.x"
  },
```

I have both the settings>general node version changed and the above in my package.json and am still getting the error. Is how I've specified engines incorrect?

**Reply by swarnava** [reply]
Yes this looks correct. Do you have any third party builder which may be using old Node engine? You can typically see them in `vercel.json`. Ideally setting `18.x` in package.json should override everything.
